Kentico/gatsby-source-kontent
# Schema definition

### Motivation

Fixes #59 
Internal ID DEVREL-323

### Checklist


### How to test

Try to generate schema with `gatsby develop`, check generated schema. Schema should be generated for types without item representation as well.
